src/native/corehost/runtime_config.cpp

```diff
@@ -359,23 +371,11 @@ bool runtime_config_t::ensure_dev_config_parsed()
 
 bool runtime_config_t::read_framework_array(const json_parser_t::value_t& frameworks_json, fx_reference_vector_t& frameworks_out, bool name_and_version_only)
 {
-    bool rc = true;
-
     for (const auto& fx_json : frameworks_json.GetArray())
     {
         fx_reference_t fx_out;
-        rc = parse_framework(fx_json, fx_out, name_and_version_only);
-        if (!rc)
-        {
-            break;
-        }
-
-        if (fx_out.get_fx_name().length() == 0)
-        {
-            trace::verbose(_X("No framework name specified."));
-            rc = false;
-            break;
-        }
+        if (!parse_framework(fx_json, fx_out, name_and_version_only))
```

Comment:
Nit: I would expect this API to have `fx_out` as the last param and the `name_and_version_only` be grouped with the "in" parameters. I see this is also done in this function, so I guess "okay". Ideally at some point in the future this could be changed?

Reply:
`name_and_version_only` is optional, so I think the current order makes sense.

Whether it makes sense for `name_and_version_only` to be optional is another matter 🙂